GooglePlacesAutocomplete.js

```diff
@@ -952,7 +952,7 @@ GooglePlacesAutocomplete.defaultProps = {
   placeholderTextColor: '#A8A8A8',
   isRowScrollable: true,
   underlineColorAndroid: 'transparent',
-  returnKeyType: 'default',
+  returnKeyType: 'search',
```

Comment:
'default' returns an error on the web. Search makes more sense anyways.  The RNW [docs](https://necolas.github.io/react-native-web/docs/?path=/docs/components-textinput--auto-capitalize#returnkeytype) don't show default as a property as per the HTML [Spec](https://html.spec.whatwg.org/dev/interaction.html#attr-enterkeyhint).